### PR TITLE
only suppress file changed notifications for C# files

### PR DESCRIPTION
**Pull request by filipw**
This is a follow up to https://github.com/OmniSharp/omnisharp-vscode/pull/4178

We should only suppress events for C# code files, not for files such as `.csproj`, `.editorconfig` or assets files, as it causes OmniSharp to go stale after changes are done in those files.

### CODE REVIEW

File: src/features/changeForwarding.ts [hunk 1]
```diff
@@ -82,6 +84,11 @@ function forwardFileChanges(server: OmniSharpServer): IDisposable {
         };
     }
 
+    function isCSharpCodeFile(uri: Uri) : Boolean {
+        let normalized = uri.path.toLocaleLowerCase();
```
Comment (333fred) [inline src/features/changeForwarding.ts:88]: Could be const

File: src/features/changeForwarding.ts [hunk 2]
```diff
@@ -69,6 +69,8 @@ function forwardFileChanges(server: OmniSharpServer): IDisposable {
                     // being that the file is now in an inconsistent state.
                     // If the document is closed, however, it will no longer be synchronized, so the text change will
                     // not be triggered and we should tell the server to reread from the disk.
+                    // This applies to C# code files only, not other files significant for OmniSharp
+                    // e.g. csproj or editorconfig files
```
Comment (333fred) [inline src/features/changeForwarding.ts:73]: Do we not get incremental updates for these files as well that we then send to the server?

Comment (filipw) [reply src/features/changeForwarding.ts:73]: at the moment there are quite a few things that need to fire as callbacks in response to file changed events coming via the files changed endpoint (https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp.MSBuild/ProjectManager.cs#L379-L477). 
At the moment none of them are arriving

**Review by 333fred — APPROVED** [review]
LGTM, one const suggestion.

**Review by JoeRobich — APPROVED** [review]

**Pull request merged**